elliptic-curve/src/point/non_identity.rs

```diff
@@ -72,6 +72,16 @@ where
             point: self.point.to_affine(),
         }
     }
+
+    /// Multiply by the generator of the prime-order subgroup.
+    pub fn mul_by_generator<C: CurveArithmetic>(scalar: NonZeroScalar<C>) -> Self
```

Comment:
Perhaps borrow here since the function you're invoking also borrows?

Reply:
Done!